App/app/controller/StoreLoadController.js

```diff
@@ -23,6 +23,8 @@ Ext.define('Enif.controller.StoreLoadController', {
         // clear the filters on the store before loading
         gamesStore.clearFilter();
```

Comment:
I know about that and I don't know, the comment says clear the filter before loading, I didn't wanted to investigate further